packages/viz/src/components/common/stat/Stat.css

```
@@ -245,3 +245,40 @@ svg.statWheelTimeInAuto .onEllipse {
 svg.statWheelTimeInAuto .offEllipse {
   fill: var(--basal, rgb(168, 200, 212));
 }
+
+.nobar {
+  display: grid;
+  row-gap: 4px;
+  margin-bottom: 0.5em;
+}
+.nobar .nobarRowTitle {
+  color: var(--text-medium-contrast);
+}
+.nobar .nobarRowValue {
+  border-width: 2px;
+  border-color: var(--stat-disabled);
+  border-style: solid;
+  border-radius: 12px;
+  text-align: center;
+  font-size: 12px;
```

I would make the row values a little bigger
```suggestion
  border-radius: 14px;
  text-align: center;
  font-size: 14px;
```